products/compute/terraform.yaml

```diff
@@ -1577,6 +1580,10 @@ overrides: !ruby/object:Overrides::ResourceOverrides
           template_name: "my-template"
           igm_name: "my-rigm"
     properties:
+      region: !ruby/object:Overrides::Terraform::PropertyOverride
+        required: false
+        ignore_read: true
+        default_from_api: true
```

Comment:
I would expect `default_from_api` to conflict with `ignore_read` - if the api provides a default but we ignore reading it, will it be available locally? This also seems to be different than the treatment in https://github.com/GoogleCloudPlatform/magic-modules/pull/4394 which moved things from `ignore_read` _to_ `default_from_api` - is the difference intentional?

Reply:
Not sure. It seems to work though.